File: asn/asn_3gpp/Java/ASN_3gppLexerBase.java
import org.antlr.v4.runtime.CharStream;
import org.antlr.v4.runtime.Lexer;

public abstract class ASN_3gppLexerBase extends Lexer {

    protected ASN_3gppLexerBase(CharStream input) {
        super(input);
    }

    protected boolean IsColumnZero()
    {
        return this.getCharPositionInLine() == 0;
    }
}

File: asn/asn_3gpp/Antlr4ng/ASN_3gppLexerBase.ts
import { CharStream, Lexer, Token } from "antlr4ng";

export default abstract class ASN_3gppLexerBase extends Lexer {
    constructor(input: CharStream) {
        super(input);
    }

    IsColumnZero(): boolean {
	return this.column == 0;
    }
}
